webpack.config.js

```diff
@@ -110,6 +111,42 @@ var wpconfig = {
                 use: 'babel-loader'
             },
             {
+                test: /\.(js|jsx)$/,
+                include: [
+                    /studio-frontend/,
+                    /paragon/
+                ],
+                use: 'babel-loader'
+            },
+            {
+                test: /.scss$/,
+                include: [
+                    /studio-frontend/,
+                    /paragon/
+                ],
+                use: [
+                    'style-loader',
```

Comment:
This is the right dev configuration for style-loader, but in production typically you'd want to extract the CSS out into a separate file ([docs on why available here](https://github.com/webpack-contrib/sass-loader#in-production)). Basically, style-loader injects a bunch of `<style>` tags into the `<head>` of the page on render, which can lead to a flash of unstyled content (and isn't super great for performance), so it's preferable to extract the CSS out into its own separately served file in production, so it can be cached.

To do this I'd recommend adding `ExtractTextPlugin` [s it's used in SFE's prod config](https://github.com/edx/studio-frontend/blob/master/config/webpack.prod.config.js#L24-L49) and gating style-loader on `isProd`. I actually implemented pretty much this exact functionality in platform in my unmerged paragon integration PR ([extractText config](https://github.com/edx/edx-platform/blob/f336cb5516842eb11fd07d722622336b04dcb5cf/webpack.config.js#L18-L21) and [loader config](https://github.com/edx/edx-platform/blob/f336cb5516842eb11fd07d722622336b04dcb5cf/webpack.config.js#L140-L162)) and it worked just fine both in dev and on a sandbox. Happy to assist with setting this up if needed.

Reply:
Yeah, I was trying to get `ExtractTextPlugin` working at one point, but got confounded by prod and dev sharing a config.

I'll attempt to get this fixed up (may ping you with questions as I go)